lib/server.js

```diff
@@ -327,16 +332,17 @@ class Server extends KarmaEventEmitter {
         done(code || 0)
       }
 
-      this.emitAsync('exit').then(() => {
+      return this.emitAsync('exit').then(() => new Promise((resolve, reject) => {
```

Comment:
I think this would be clearer with an ArrowFunction 
```
...() => {
      return new Promise...
```

Reply:
I'm sorry, I don't understand. This is already an arrow function.

Do you mean adding the block to the arrow function instead of the return value directly?

Reply:
Yes a function rather than an ArrowExpression

Reply:
Oh I guess it is called a concise body vs a block body
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#Function_body

Reply:
Got it, changed.